[2022-06-12 16:51:18,119] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T02:18:00+00:00 [queued]>
[2022-06-12 16:51:18,160] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T02:18:00+00:00 [queued]>
[2022-06-12 16:51:18,161] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:51:18,161] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-12 16:51:18,162] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:51:18,233] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-12 02:18:00+00:00
[2022-06-12 16:51:18,239] {standard_task_runner.py:52} INFO - Started process 5818 to run task
[2022-06-12 16:51:18,246] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-12T02:18:00+00:00', '--job-id', '672', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpbtt60e1s', '--error-file', '/tmp/tmp0txewirw']
[2022-06-12 16:51:18,247] {standard_task_runner.py:80} INFO - Job 672: Subtask save_data_task
[2022-06-12 16:51:18,382] {task_command.py:370} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T02:18:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-12 16:51:18,863] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T02:18:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T02:18:00+00:00
[2022-06-12 16:51:18,881] {python.py:173} INFO - Done. Returned value was: None
[2022-06-12 16:51:18,910] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220612T021800, start_date=20220612T145118, end_date=20220612T145118
[2022-06-12 16:51:18,983] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-12 16:51:19,195] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
